services/ad-tech/src/lib/attribution-reporting-helper.ts

```diff
@@ -99,7 +99,7 @@ export const handleAttributionSourceRegistration = (
       .filter(([key, _]) => key !== 'redirect')
       .map(([key, value]) => `${key}=${value}`)
       .join('&');
-    const redirectUrl = `${req.query['redirect']}/register-source?${query}`;
+    const redirectUrl = `${req.query['redirect']}/attribution/register-source?${query}`;
```

Comment:
Should this be `attribution-reporting` or is it different than the path to [where reports are sent](https://github.com/privacysandbox/aggregation-service/blob/main/docs/collecting.md)?